pages/learn/accounts/enterprise-sso.md

```diff
@@ -46,43 +62,47 @@ __Note__: If you unlink the default team in the Identity Provider configuration,
 * Data and Privacy: Your personal data associated with the account will be transferred to the company. The company will be responsible for the protection and use of your data in accordance with the existing privacy policy.
 * Consent: By proceeding with this transfer, you confirm that you have the authority to transfer the account to the company and that you consent to the changes outlined above.
 
-By clicking the "Enable" button, you agree to the above terms. If you have any questions or concerns, please contact our support team before completing the transfer.
+By clicking the Enable button, you agree to the above terms. If you have any questions or concerns, please contact our support team before completing the transfer.
 
 ### Log in with a SAML account
 
-Once you have enabled SAML on your account, you can log in by following the "Enterprise SSO" login button or the login URL provided by your balenaCloud organization administrator, e.g., `https://dashboard.balena-cloud.com/saml/acme`.
+Once you have enabled SAML on your account, you can log in using the Enterprise SSO login button or the login URL provided by your balenaCloud organization administrator, e.g., `https://dashboard.balena-cloud.com/saml/acme`.
+
 <!-- TODO: Update to latest screenshot -->
 <img alt="Login page with Enterprise SSO login highlighted." src="/img/common/saml/login-page.png" width="60%">
 
-Once you have enabled SAML, you can no longer log in using a `password` and should always use the SSO login method.
+Once you have enabled SAML, you can no longer log in using a `username` and `password` combination and you must always use the SSO login method.
 
 ## FAQs
 
 #### How do I set up an Identity Provider?
 Each SAML Identity Provider (IdP) has its own unique implementation and terminology, which can result in variations in the configuration process. While it is not feasible to provide detailed configuration guidelines for every IdP, we have included example guides for two of the major providers: [Microsoft Entra ID (formerly Azure Active Directory)][ms-saml] and [Google Workspace SAML](/learn/accounts/idp-setup/google-workspace-saml-setup/). These examples are designed to help you understand the necessary steps and how to configure your own provider effectively.
 
+#### Can I use any identity provider with balena’s Enterprise SSO?
+We support any identity provider as long as they are compliant with the SAML 2.0 protocol specification such as Okta, [Microsoft Entra ID][ms-saml] (formerly Azure AD) and [Google Workspace][google-saml]. We have provided examples for using Microsoft Entra ID and Google Workspace in our documentation. To setup your IdP, get the Federation Metadata XML as we have setup for Microsoft/Google IdP's and then follow the [instructions to link that IdP](https://docs.balena.io/learn/accounts/enterprise-sso/#link-a-saml-identity-provider) to balenaCloud's SSO.
```

Comment:
Good stuff adding this new FAQ entry, and linking the examples. I think, however, that the links should be on the sentence that starts with “We have provided examples …” instead.

Reply:
Made the line better, and leaner instead.